modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -431,3 +478,30 @@ def _apply_list_of_funcs(funcs, partition):  # pragma: no cover
         len(partition.columns) if hasattr(partition, "columns") else 0,
         get_node_ip_address(),
     )
+
+
+@ray.remote(num_returns=4)
```

Comment:
qq: why do we need to tell ray the number of things we are returning? Is this an optimization?

Reply:
`num_returns=4` gives 4 output object IDs instead of 1 object ID with a reference to a 4-tuple. e.g. try

```python
@ray.remote
def f():
  return 1, 2
print(ray.get(f.remote()))
```

and get `(1, 2)`

but 

```python
@ray.remote(num_returns=2)
def f():
  return 1, 2
print(f.remote())
```

gives 2 object ids, and if we `ray.get` each one, we get 1 and 2 respectively.

ref. https://docs.ray.io/en/latest/ray-core/package-ref.html#ray-wait

Reply:
I believe this tells Ray how many object references to expect to return. If we didn't, it would probably default to 1 and give us one object ID pointing to a tuple, which we may not want (e.g. in the split case where we want to materialize certain splits in different places).